tests/Microsoft.Identity.Test.Unit/RequestsTests/SilentRequestTests.cs

```diff
@@ -156,13 +133,27 @@ public async Task BrokerSilentRequestTestExecutorAsync(bool brokerConfiguredByUs
 
                 var request = new SilentRequest(harness.ServiceBundle, parameters, silentParameters);
 
-                var result = await request.RunAsync(default).ConfigureAwait(false);
+                if (brokerConfiguredByUser) //When broker is enabled, local cache should not be used
```

Comment:
Is it worth splitting this 4 data row test into 2 tests? I think they would be more readable.